README.md

```diff
@@ -33,16 +33,22 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
+   Semantic HTML gives meaning to a web page.
```

Comment:
this answer is good but try being more specific. Semantic HTML gives
 **specific**  meaning to sections of a web page.